### PR TITLE
ci(HMS-3962): Update Integration test image for IQE

### PR DESCRIPTION
**Pull request by akhil-jha**
Name update from hms-integration to insights-experiences.

### CODE REVIEW

**Comment by akhil-jha** [comment]
@jrusz

File: deploy/iqe-trigger-integration.yml [hunk 1]
```diff
@@ -117,13 +117,16 @@ parameters:
   from: "[a-z0-9]{6}"
 - name: IQE_IMAGE
   description: "container image path for the iqe plugin"
-  value: quay.io/cloudservices/iqe-tests:hms-integration
+  value: quay.io/cloudservices/iqe-tests
```
Comment (jrusz) [inline deploy/iqe-trigger-integration.yml:120]:
```suggestion
  value: quay.io/cloudservices/iqe-tests:insights-experiences
```
Same as in image-builder repo, let's add the tag here.

**Review by jrusz — CHANGES_REQUESTED** [review]
One small change requested + please add a description to you commit to make the "best practices" check happy.

**Review by jrusz — APPROVED** [review]

**Pull request merged**